### PR TITLE
Add Honor Mobile Map Package Expiration Date sample

### PR DESCRIPTION
**Pull request by philium**

### CODE REVIEW

File: arcgis-ios-sdk-samples/Maps/Honor mobile map package expiration date/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - Use `AGSExpiration.dateTime` to get the expiration date set by the author of the MMPK.
+
+## Relevant API
+
+- `AGSExpiration`
+- `AGSMobileMapPackage`
+
+## Offline Data
+
+This sample uses the [LothianRiversAnno - Expired](https://arcgisruntime.maps.arcgis.com/home/item.html?id=628e8e3521cf45e9a28a12fe10c02c4d) Mobile Map Package. It is downloaded from ArcGIS Online automatically.
```
Comment (saratchandrakarumuri) [inline arcgis-ios-sdk-samples/Maps/Honor mobile map package expiration date/README.md]: Wrong portal item is referred here. This link redirects to a tpk file.

Comment (philium) [reply arcgis-ios-sdk-samples/Maps/Honor mobile map package expiration date/README.md]: 🤦🏼‍♂️

Comment (philium) [reply arcgis-ios-sdk-samples/Maps/Honor mobile map package expiration date/README.md]: Thank you for finding that!

**Comment by philium** [comment]
Thank you @njarecha and @saratchandrakarumuri!

**Pull request merged**